app/views/users/index.html.erb

```diff
@@ -0,0 +1,19 @@
+<div class="container">
+
+  <h1 class="text-center">All Users</h1>
+<header>
+<% if current_user %>
+  <h3 class="navbar-brand">Hello <%= current_user.name %></h3>
+<% else %>
+  <h3 class="navbar-brand">No User Signed In</h3>
+<% end %>
```

Comment:
- [ ] create a helper method that has the if/else logic and render the links and markup within the helper method instead of doing it straight in the view